Core/src/org/sleuthkit/autopsy/casemodule/CaseDetails.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2017 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.casemodule;
+
+/**
+ * Wrapper to contain the modifiable details of a case, such as case display
+ * name, case number, and examiner related fields.
+ */
+final class CaseDetails {
+
+    private final String caseDisplayName;
+    private final String caseNumber;
+    private final String examinerName;
+    private final String examinerPhone;
+    private final String examinerEmail;
+    private final String caseNotes;
+
+    CaseDetails(String displayName, String number, String exName, String exPhone, String exEmail, String notes) {
```

Comment:
All of the fields in a CaseDetails object except the case display name are optional. For clarity, I would like to see us model this by providing a constructor that takes only a display name argument, setting all other fields to the default values of the empty string.

We can then use this in AutoIngestManager to call the new Case.createAsCurrentCase method overload (see prior comments) cleanly and compactly; this will also be nice for the large number of Case.createAsCurrentCase calls we will soon be seeing in unit test code.